server/controllers/PaymentRequest.js

```diff
@@ -53,7 +53,7 @@ class PaymentRequest {
       extraPayload: req.body.extraPayload,
       timeStamp: req.timeStamp,
       encryptedPassword: req.encryptedPassword,
-      callbackURL: `${req.protocol}://${req.hostname}/api/v${process.env.API_VERSION}/payment/success`,
+      callbackURL: `${req.protocol}://${req.hostname}:8080/api/v${process.env.API_VERSION}/payment/success`,
```

Comment:
What you want is to set the port dynamically and not hardcode it into the string literal. Not everyone is using port `8080`. This would break everyone's build if they were to pull this update after merging.

I also think the `${req.hostname}` takes care of this. Not so sure though.